apps/redis-sentinel/templates/services/app/svc.yml.j2

```diff
@@ -59,6 +59,7 @@ spec:
       - "{{ config }}"
 {% endfor %}
 {% endif %}
+{% if redis_sentinel_redis_persistence_enabled %}
```

Comment:
WDYT?
```suggestion
{% if redis_sentinel_redis_persistence_is_enabled %}
```

Reply:
I'm not a big fan of the `is` suffix, `enabled` is enough for me but why not.

Reply:
ok then